2021-12-05 17:20:40,700 INFO    Thread-8  :20334 [wandb_setup.py:_flush():69] setting env: {'project': 'NMT', 'entity': 'jungwoohan72', 'root_dir': '/home/jungwoo/Coursework/AI504/Project2', 'run_id': '79u890n1', 'sweep_param_path': '/home/jungwoo/Coursework/AI504/Project2/wandb/sweep-xo69487m/config-79u890n1.yaml', 'sweep_id': 'xo69487m'}
2021-12-05 17:20:40,700 INFO    Thread-8  :20334 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-05 17:20:40,700 INFO    Thread-8  :20334 [wandb_init.py:_log_setup():342] Logging user logs to /home/jungwoo/Coursework/AI504/Project2/wandb/run-20211205_172040-79u890n1/logs/debug.log
2021-12-05 17:20:40,700 INFO    Thread-8  :20334 [wandb_init.py:_log_setup():343] Logging internal logs to /home/jungwoo/Coursework/AI504/Project2/wandb/run-20211205_172040-79u890n1/logs/debug-internal.log
2021-12-05 17:20:40,700 INFO    Thread-8  :20334 [wandb_init.py:init():375] calling init triggers
2021-12-05 17:20:40,700 INFO    Thread-8  :20334 [wandb_init.py:init():380] wandb.init called with sweep_config: {'attention_heads': 4, 'batch_size': 64, 'decoder_layers': 3, 'dropout': 0.2, 'emb_dim': 32, 'encoder_layers': 3, 'ffn_dim': 32, 'lr': 0.0001, 'nepochs': 50, 'patience': 15}
config: {}
2021-12-05 17:20:40,701 INFO    Thread-8  :20334 [wandb_init.py:init():424] starting backend
2021-12-05 17:20:40,701 INFO    Thread-8  :20334 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-05 17:20:40,704 INFO    Thread-8  :20334 [backend.py:ensure_launched():135] starting backend process...
2021-12-05 17:20:40,736 INFO    Thread-8  :20334 [backend.py:ensure_launched():139] started backend process with pid: 20470
2021-12-05 17:20:40,737 INFO    Thread-8  :20334 [wandb_init.py:init():429] backend started and connected
2021-12-05 17:20:40,743 INFO    Thread-8  :20334 [wandb_run.py:_config_callback():826] config_cb None None {'attention_heads': 4, 'batch_size': 64, 'decoder_layers': 3, 'dropout': 0.2, 'emb_dim': 32, 'encoder_layers': 3, 'ffn_dim': 32, 'lr': 0.0001, 'nepochs': 50, 'patience': 15}
2021-12-05 17:20:40,744 INFO    Thread-8  :20334 [wandb_init.py:init():477] updated telemetry
2021-12-05 17:20:40,745 INFO    Thread-8  :20334 [wandb_init.py:init():500] communicating current version
2021-12-05 17:20:41,295 INFO    Thread-8  :20334 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-05 17:20:41,295 INFO    Thread-8  :20334 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-12-05 17:20:41,525 INFO    Thread-8  :20334 [wandb_init.py:init():540] starting run threads in backend
2021-12-05 17:20:43,607 ERROR   Thread-8  :20334 [wandb_init.py:init():801] error
Traceback (most recent call last):
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 775, in init
    run = wi.init()
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 543, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 801, in communicate_run_start
    result = self._communicate(rec)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 539, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 77, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
Exception
